src/ansys/fluent/visualization/contour.py

```diff
@@ -0,0 +1,52 @@
+from typing import List, Optional
+
+
+class Contour:
+    def error_check(self, solver):
```

Comment:
I guess this is intended to be private? `_error_check`?

Reply:
Done. Thank you.